Automattic/wp-calypso
# Jetpack Manage: Adding a new Manage Pricing page

Related to #https://github.com/Automattic/jetpack-manage/issues/244

## Proposed Changes

* This PR adds a new Jetpack Manage pricing page at `/manage/pricing`.
* Once this PR is merged the page will only be available in staging and development.
* The page includes a tracking event on page view
* The page also includes the Jetpack.com Masterbar, but the main menu is not sticky unlike on other Jetpack.com pages (because the license selector will be sticky, a later addition).
* Currently the page also includes a header with a sub-heading, a footer, recommendations block and FAQ section, as with the current `cloud.jetpack.com/pricing` page.
* There is no other content at this stage.
* This PR also includes a change to one of the paths defined for the `cloud.jetpack.com/pricing` page in `client/sections.js`, to make sure content from the pricing page doesn't appear on the `/manage/pricing` page.



## Testing Instructions

* To test, check out this PR and visit `http://jetpack.cloud.localhost:3000/manage/pricing`. What you see should be similar to the below screenshot.
* You should also see a `calypso_jetpack_manage_pricing_visit` firing.


<img width="458" alt="Screenshot 2024-01-23 at 16 58 35" src="https://github.com/Automattic/wp-calypso/assets/16754605/320072cd-8946-4af4-9def-9d2ec5a59f82">


## Pre-merge Checklist
